lib/thumbnail-sizes.php

```diff
@@ -25,6 +25,8 @@
   add_image_size( 'l-8-square', 866, 866, true );
   add_image_size( 'l-8-full', 866, 9999, false );
 
+  add_image_size( 'lightbox', 1030, 644, false );
```

Comment:
Ok lets see how it is but surely the oversize should be solvable with CSS? I think there is a class that I styled for the thumbnail that could be used?